lib/Optimizer/Transforms/LoopAnalysis.cpp

```diff
@@ -258,7 +269,7 @@ bool opt::isaMonotonicLoop(Operation *op, LoopComponents *lcp) {
     // This is a `for` loop and must have a body with a continue terminator.
     // Currently, only a single basic block is allowed to keep things simple.
     // This is in keeping with our definition of structured control flow.
```

Comment:
Is this comment still true? I would expect `if (...) { ...; break; }` code to produce multiple basic blocks.

Reply:
No.